docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,114 @@
+=============
+Accessibility
+=============
+
+All aspects of Django must be compatible with our accessibility standards,
+and with the specific support targets we aim for.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+Django aims for conformance with `WCAG 2.2`_ at the AA level.
```

Comment:
"aims" feels like a bit of a cop-out, I'd prefer stronger language here.

Reply:
I’ll try!

Reply:
There's a tension that's hard to rectify: We definitely want to use strong language, but we also have to be honest about the fact that we're not there yet. In my suggestion on the intro above, I chose "We work … to …" as wording that indicates we are actively doing something, not just aiming/aspiring/seeking. Perhaps that sort of change would be good here, as well.

Reply:
I’ve tried to make this stronger, without making it sound like we’re there yet.